packages/react-components/src/components/MessageThread.tsx

```diff
@@ -204,22 +204,28 @@ const DefaultSystemMessageRenderer: DefaultMessageRendererType = (props: Message
       );
     }
     if (payload.type === 'participantAdded') {
-      return (
-        <SystemMessageComponent
-          iconName={(payload.iconName ?? '') as SystemMessageIconTypes}
-          content={generateParticipantsStr(payload.participants) + strings.messageThread.participantJoined}
-          containerStyle={props?.messageContainerStyle}
-        />
-      );
+      const participantsStr = generateParticipantsStr(payload.participants);
+      if (participantsStr !== '') {
```

Comment:
nit: can you add a comment here that this is here to filter out _our_ moderator that has no display name